test/package.json

```diff
@@ -5,6 +5,7 @@
   "private": true,
   "scripts": {
     "test": "node --preserve-symlinks test.js test",
+    "debug": "node --preserve-symlinks test.js test --log-level debug",
```

Comment:
My other similar comment applies here too: https://github.com/digitalbazaar/bedrock-web-profile-manager/pull/81/files#r675933035
This is change making my point.  I think we should find a solution that doesn't require adding this to every project.